src/main/java/baseball/ResultTell.java

```diff
@@ -0,0 +1,27 @@
+package baseball;
+
+import java.util.List;
+
+public class ResultTell {
+
+    public static String HowResult(int a, int num, List computerList, List playerList) {
+        String result = "";
+        int strike = 0;
+        if (a == 0) {
+            return "낫싱";
+        }
+        for (int i = 0; i < num; i++) {
+            if (playerList.get(i) == computerList.get(i)) {
```

Comment:
객체간의 비교는 equls 메서드를 이용합니다.
```suggestion
            if (playerList.get(i).equals(computerList.get(i))) {
```